src/adapter.ts

```diff
@@ -143,6 +143,19 @@ export default class TypeORMAdapter implements Adapter {
         await getRepository(this.getCasbinRuleConstructor(), this.option.name).save(line);
     }
 
+    /**
+     * addPolicies adds policy rules to the storage.
+     */
+    public async addPolicies(sec: string, ptype: string, rules: string[][]) {
+        const lines: GenericCasbinRule[] = [];
+        for (const rule of rules) {
+            const line = this.savePolicyLine(ptype, rule);
+            lines.push(line);
+        }
+
+        await getRepository(this.getCasbinRuleConstructor(), this.option.name).save(lines);
```

Comment:
It is important to note that most databases have the maximum batch size, If the batch size is too large will fail.

I'm not sure the TypeORM how to deal with.